packages/internals/src/__tests__/engine-commands/__snapshots__/formatSchema.test.ts.snap

```diff
@@ -41,7 +41,8 @@ model Like {
   post   Post   @relation(fields: [postId], references: [id])
 
   @@unique([userId, postId])
-}"
+}
+"
```

Comment:
So we have a new line here?

Maybe we should remove the added `os.EOL` line 66, so we keep the same output as before

Reply:
Removed that `os.EOL` line.

Reply:
Ok, good to know as it will impact all users of the command but makes sense to do the same as what the extension does 👍🏼